### PR TITLE
Irregular release work ( ~ December, 2017)

### PR DESCRIPTION
**Pull request by daisukenishino2**
Release of 02-13 version (within December 2017)

### CODE REVIEW

File: Readme.ja.md [hunk 1]
```diff
@@ -4,12 +4,18 @@
...
+- IDE（統合開発環境）
   - Visual Studio 2015
-- 実行環境
-  - .NET Framework 4.6
+- targetFramework（実行環境）
+  - .NET Framework 4.5.2 (net452)
+  - .NET Framework 4.6 (net46)
+  - .NET Framework 4.7 (net47)
+
+既定のプロジェクト・ソリューションは、net46をtargetFrameworkにしています。
```
Comment (yusukemaegawa) [inline Readme.ja.md]: @daisukenishino 

プロジェクト・ソリューションの既定の targetFramework は net46(.NET Framework 4.6) です。
その他の targetFramework 向けのプロジェクト・ソリューションには、それぞれ targetFramework がプロジェクト・ソリューション名に含まれます。
たとえば、net47(.NET Framework 4.7) を対象としたプロジェクト・ソリューションは、"{identifier}_net47.{ext}" と命名しています。

File: README.md [hunk 2]
```diff
@@ -4,12 +4,18 @@
...
+- IDE (Integrated Development Environment)
   - Visual Studio 2015
-- Run-time environment
-  - .NET Framework 4.6
+- targetFramework (Run-time environment)
+  - .NET Framework 4.5.2 (net452)
+  - .NET Framework 4.6 (net46)
+  - .NET Framework 4.7 (net47)
+
+The default project and solution is set to net46 targetFramework.
```
Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

The default targetFramework of projects and solutions is net46(.NET Framework 4.6).
The name of projects and solutions for other targetFramework include the targetFramework.
For example, the projects and solutions for net47(.NET Framework 4.7) are named *{identifier}_net47.{ext}*.

**Comment by daisukenishino2** [comment]
@yusukemaegawa 

I have made a correction to your pointed out.
Please merge for this pull request after confirming.

**Comment by daisukenishino** [comment]
@yusukemaegawa

I have made a correction to your pointed out.
Please merge for this pull request after confirming.

File: root/programs/C#/Frameworks/Infrastructure/Framework/Authentication/JwtToken.cs [hunk 3]
```diff
@@ -99,16 +101,23 @@ public class JwtToken
                     scopes = JsonConvert.DeserializeObject<List<string>>(jobj["scopes"].ToString());
                 }
 
+                long unixTimeSeconds = 0;
+#if NET45
```
Comment (yusukemaegawa) [inline root/programs/C#/Frameworks/Infrastructure/Framework/Authentication/JwtToken.cs]: @daisukenishino 

Where is this symbol defined?

Comment (daisukenishino2) [reply root/programs/C#/Frameworks/Infrastructure/Framework/Authentication/JwtToken.cs]: @yusukemaegawa 

See : https://github.com/OpenTouryoProject/OpenTouryo/blob/develop/root/programs/C%23/Frameworks/Infrastructure/Framework/Framework_net45.csproj#L50

**Pull request merged**